README.md

```diff
@@ -99,6 +99,16 @@ EOF
 kubectl get vmimports example-virtualmachineimport -n default
 ```
 
+### Wait for VM Import resource to complete:
+```bash
+kubectl wait vmimports example-virtualmachineimport -n default --for=condition=Succeeded
+```
+
+### View vm-import logs:
+```bash
+kubectl logs -n kubevirt-hyperconverged vm-import-controller-86cc9c9dcd-l65fk
```

Comment:
Wouldn't it be better to add something like `kubectl logs -n kubevirt-hyperconverged deploy/vm-import-controller`, it's more universal, then pod name.